enumfields/fields.py

```diff
@@ -32,7 +32,7 @@ def to_python(self, value):
         raise ValidationError('%s is not a valid value for enum %s' % (value, self.enum), code="invalid_enum_value")
 
     def get_prep_value(self, value):
-        return None if value is None else value.value
+        return None if value is None else value
```

Comment:
I did a PR before seeing this one. Though the solution is slightly different. `get_prep_value` instead of returning the `value` it performs a cast. See https://github.com/hzdg/django-enumfields/pull/46/files#diff-0d0002b8d572eae99755278bc4aeb328R35